SPEC.md

```diff
@@ -1274,6 +1335,54 @@ workflow wf {
 
 In this example, the fully-qualified names that would be exposed as workflow outputs would be `wf.task1.results`, `wf.altname.value`.
 
+# Sub Workflows
+
+Workflows can also be called inside of workflows, like a regular call.
```

Comment:
I think this phrasing is an artifact of this being a spiffy new feature. I'd phrase this as if it *is* a regular call, just a workflow instead of a task